strapi/strapi
# [TS] Update kitchensink-ts to the latest beta

### What does it do?

Update the kitchensink-ts app to the latest Typescript support beta.

### Why is it needed?

This application was created using the first TS support beta and is not working anymore.

### How to test it?

- Clone the repository
- Navigate to `examples/kitchensink-ts`
- Run yarn develop
- Everything should work and strapi should start
